tests/pytorch/test_pytorch_model_export.py

```diff
@@ -215,29 +215,18 @@ def test_signature_and_examples_are_saved_correctly(sequential_model, data):
 @pytest.mark.large
 @pytest.mark.parametrize("scripted_model", [True, False])
 def test_log_model(sequential_model, data, sequential_predicted):
-    old_uri = tracking.get_tracking_uri()
-    # should_start_run tests whether or not calling log_model() automatically starts a run.
-    for should_start_run in [False, True]:
```

Comment:
The behavior tested by this case can be captured by simplifying to `should_start_run=False`.